### ISSUE TITLE
Tutorial on how to initialize model with the dirty image

### ISSUE THREAD

**Issue by iancze** [MPoL-dev/MPoL#35]
**Is your feature request related to a problem or opportunity? Please describe.**
An RML forward model can be optimized from any starting point, but it will make the optimization that much quicker if the starting point already has a low loss value (maximal likelihood). 

One such (easy-to-calculate) image is the dirty image, which already has a maximal likelihood value. However, the dirty image usually contains regions of negative flux, which are not allowed by some model parameterizations (e.g., [BaseCube w/ default Softplus pixel mapping](https://github.com/MPoL-dev/MPoL/blob/main/src/mpol/images.py#L13), which enforces image positivity).

**Describe the solution you'd like**
One way around this is to optimize the model *based off of the image-plane distance* between the dirty image and the state of the ImageCube. The pixel-to-pixel L2 norm seems like a reasonable distance metric in this case.

```
# calculate the dirty image we want to optimize to
# doesn't need to be uniform gridding in this case
# since we aren't using data visibilities just yet
gridder.grid_visibilities(weighting="briggs", robust=0.0)
img = gridder.get_dirty_image("Jy/arcsec^2")
# make img a tensor
dirty_image = torch.tensor(img.copy())

for iteration in range(500):

    optimizer.zero_grad()

    model.forward()
    sky_cube = model.icube.sky_cube

    loss = loss_fn(sky_cube, dirty_image)

    loss.backward()
    optimizer.step()

torch.save(model.state_dict(), "dirty_image_model.pt")
```

The model state can  be saved, and then reloaded right before optimizing the model against the visibility data.

```
model.load_state_dict(torch.load("dirty_image_model.pt"))

# be sure to reset gridded visibilities to "uniform" before optimizing 
gridder.grid_visibilities(weighting="uniform")
dataset = gridder.to_pytorch_dataset()
```

This could be documented as a short, self-contained tutorial, that can be referenced from other tutorials that go all the way to optimizing directly against visibility data.

In short tests so far, this pseudo-dirty image provides a much better optimization starting point than a blank image, which means the learning rate of the optimizer can be tuned more aggressively to reach convergence.

The tutorial should also be clear about the fact that RML optimization doesn't *need to* start from the dirty image, and therefore isn't really a deconvolution algorithm in the same sense as `tclean`.
